{"name": "Проверка на ошибку при использовании неверных данных", "status": "passed", "description": "Проверяем, что пользователь с некорректными данными не может авторизоваться", "start": 1724608145975, "stop": 1724608146238, "uuid": "4efedade-808e-4658-ab28-6b2d714ce3f5", "historyId": "08634cc7e2fe52868ac90e98c3a2e9cc", "testCaseId": "08634cc7e2fe52868ac90e98c3a2e9cc", "fullName": "tests.test_login.TestLogin#test_login_incorrect_data_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-ILSU1PHQ"}, {"name": "thread", "value": "20772-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}